scripts/cicd/verify-gather-logs-operations.py

```diff
@@ -73,9 +89,15 @@ def gather_logs(command):
                         required=True, type=valid_krbid)
     parser.add_argument('-c', dest='cluster', help="Cluster name",
                         required=True, choices=VALID_CLUSTER_NAMES)
+    parser.add_argument('-n', dest='nodes', help="Nodes to gather logs from",
```

Comment:
It's using "-n node1 node2" here... I can change the argument to `action=append` for the other syntax "-n node1 -n node2" but I thought that not having to add all the "-n" would save some typing to users...